# Manual Assignment: Developing biochemical mechanism hypotheses explaining a disease phenotype	

## Lab Overview
Biochemistry is everything and has been creating life for billions of years long before there were any professors or textbooks.  If we understand the biochemical mechanisms of normal and disease states in a deep manner, we should in many cases be able to correct most aberrant phenotypes using molecular tools.  In this lab, we will mine genetic information that points to candidate epilepsy genes and their encoded biochemistry.

## Lab Objectives:
In this lab you identify candidate genes underlying a complex phenotype, epilepsy.
•	Identify genes associated with 3 epilepsy subtypes.
•	Mine for gene attributes using the genome browser.

Follow these lab instructions:

## Task A: Using OMIM to find genes associated with a disease.  
Epilepsy is a “spectrum condition with a wide range of seizure types and control varying from person-to-person” (Epilepsy Foundation). Since epilepsy is a mixture of phenotypes, there are probably many different genes involved in each phenotype etiology. Here are three subtypes of epilepsy:

* MYOCLONIC EPILEPSY OF UNVERRICHT AND LUNDBORG (OMIM # 254800)
* EPILEPSY, FAMILIAL ADULT MYOCLONIC, 2 (OMIM #607876)
* ROLANDIC EPILEPSY, MENTAL RETARDATION, AND SPEECH DYSPRAXIA, X-LINKED (OMIM #300643)

For each subtype, find the OMIM database entry for this disease at NCBI (e.g. choose the correct database using the Entrez search bar).  Read the OMIM report. For each syndrome takes notes on the clinical aspects (symptoms, age of onset, associated genes, etc.).  Summarize your findings and compare and contrast these 3 phenotypes. 

## Task B: Using the genome browser to explore gene structure.  
Make an Excel spreadsheet  where one gene associated with the phenotype are the headers.  Locate the three genes in the UCSC browser (hg38 build) and add the gene structure information rows to the spreadsheet for the longest transcript variant  in the GENECODE Track (e.g. GENECODE v48 or higher).  For each gene, record these values in the spreadsheet:

* Gene Name
* Transcription Start Coordinate
* Transcription Stop Coordinate
* Translation Start Coordinate
* Stop Coordinate
* Exon Count
* Intron Count

## Task C: Find gene annotations associated with these genes.
### Step 1. What are the molecular functions associated with these genes? 
For each of the 3 genes, click on the longest transcript version of gene in the browser to go to the Gene Description and Page Index.  Add these rows to your spreadsheet (Total counts of an annotation are fine and zero is a number).  
* Most highly expressed tissue
* Chemicals that interact with the protein from this gene from the CTD database
* Pfam protein domains
* GO Molecular Functions
* GO Biological Processes
* GO Cellular Components
* KEGG Pathways
* Reactome Pathways 

### Step 2.  What DNA variation is known for these loci? Using Click on the “Common SNPs (153)” Hyperlink.  Read about this track.  Add these data that show how many DNA polymorphisms are known for the largest transcript: 
* 5’ UTR SNPs
* 3’ UTR SNPs
* 5’ CDS SNPs (synonymous)
* 5’ CDS SNPs (nonsynonymous)

### Step 3.  Identify interacting genes.  
Search for gene products that interact with these three genes at BioGrid (http://thebiogrid.org) and Intact (http://www.ebi.ac.uk/intact ).  In your spreadsheet, indicate the total number of interacting human genes for each target gene in each database:

* BioGrid Interactions
* Intact Interacions

## Task D: Molecular mechanism hypothesis.
For each gene, construct a hypothesis as to how the altered biochemistry of that gene could lead to the phenotype.
